Makefile

```diff
@@ -1,6 +1,12 @@
+
 .PHONY: clean documentation
 
 
+ROOT_DIR:=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
```

Comment:
Root of the repo? Would `git rev-parse --show-toplevel` be more robust, in case we start including Makefiles?

Because I’d be tempted to force most of this into a separate `hypothesis-python/Makefile`, which just gets included at the top level.